server/src/resolvers/Mutation.ts

```diff
@@ -1,30 +1,31 @@
 import bcrypt from 'bcryptjs'
 import  jwt  from "jsonwebtoken"
-import {  APP_SECRET, getUserId } from "../utils.js"
+import {  APP_SECRET } from "../utils"
+import { PrismaClient } from "@prisma/client";
 
 
 export const Mutation = {
 
-    addBlog: async (parent, { input }, context) => {
+    addBlog: async (parent: any, {input} : { input:{ title:string, body:string, small:string, name:string} }, context:any) => {
```

Comment:
I don't see this type used anywhere else, but it is by default a better practice to define these as `type` or `interface`.